src/olympia/accounts/templatetags/jinja_helpers.py

```diff
@@ -1,23 +1,21 @@
-import jinja2
+from urllib.parse import quote_plus
 
 from django_jinja import library
+import jinja2
 
-from olympia.accounts import utils
+from olympia.accounts.utils import path_with_query
+from olympia.amo.templatetags.jinja_helpers import drf_url
 
 
 @library.global_function
 @jinja2.pass_context
 def login_link(context):
-    return utils.default_fxa_login_url(context['request'])
+    next_path = path_with_query(context['request'])
+    login_start = drf_url(context, 'accounts.login_start')
+    return f'{login_start}?to={quote_plus(next_path)}'
 
 
 @library.global_function
 @jinja2.pass_context
 def register_link(context):
-    return utils.default_fxa_register_url(context['request'])
-
-
-@library.global_function
-@jinja2.pass_context
-def fxa_config(context):
-    return utils.fxa_config(context['request'])
+    return login_link(context)
```

Comment:
If `register_link` just returns `login_link` now, do we need `register_link` at all? Could we just call `login_link` from anywhere we're currently calling `register_link`? If so, should it perhaps be called `fxa_link`?

Reply:
We technically don't need it anymore, I wasn't sure if we'd need the distinction. I guess I can remove it and rename the other.

Reply:
450ea6c7e8870a81750ccd50e94849541d96634c (didn't end up doing the renaming, I don't think it's that much clearer)